src/backend/core/tests/api/test_entitlements_messages.py

```
@@ -0,0 +1,123 @@
+
+pytestmark = pytest.mark.django_db
+
+
+
+
+@pytest.mark.django_db()
+@responses.activate
+def test_api_entitlements_list():
+    """Test the entitlements list endpoint."""
+
+    metrics_usage_endpoint = f"http://messages.suite.anct.gouv.fr/metrics/usage"
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)